### ISSUE TITLE
Issue on playing videos

### ISSUE THREAD

**Issue by keijiota** [jspsych/jsPsych#3231]
Dear community, 

Hello. I use a plugin video-keyboard-response and ask people make a judgment after watching a video clip. But sometimes I see that videos fail to play and it leaves a blank background. It produces an error message below on a developer tool. 
https://developer.chrome.com/blog/play-request-was-interrupted

This happens regardless of which web browser I use. 
I understand there is always a risk of not being able to play videos. But I need to take a record of it when it happens. Would you please let me know how to get this error response and write it on the data file of the excel sheet? 

Many thanks as always.
Keiji

**Comment by jodeleeuw** [comment]
Hi @keijiota 

Can you share the parameters you are using for the trial? I want to see if I can reproduce the problem and figure out a solution for the plugin.

**Comment by keijiota** [comment]
Yes, sure here is it. The problem happens more often when I run it on Gorilla. 

S0L0 = ['videos/S0L0.mp4']
var preload = {type: jsPsychPreload, 
      video: S0L0};

var FirstStim = {
  type: jsPsychVideoKeyboardResponse,
  stimulus: S0L0,
  choices: "NO_KEYS",
  response_ends_trial: true,
  trial_ends_after_video: true,
  trial_duration: 5000, 
  width: 600,
};

timeline = timeline.concat(preload); 
timeline = timeline.concat(FirstStim);

**Comment by alessandraluz** [comment]
Did you manage to get a solution for this @keijiota ? I am facing a similar issue.

**Comment by saritaraghunath** [comment]
was anyone able to solve this? running into this issue trying to play mp4 file using video keyboard response. incidentally it does not work on the jspsych demo either so not sure how to proceed
